website/docs/cloud-docs/api-docs/agents.mdx

```diff
@@ -67,6 +67,7 @@ This endpoint supports pagination [with standard URL query parameters](/cloud-do
 | `sort`         | **Optional.** Allows sorting the returned agents pools. Valid values are `"name"` and `"created-at"`. Prepending a hyphen to the sort parameter will reverse the order (e.g. `"-name"`). |
 | `page[number]` | **Optional.** If omitted, the endpoint will return the first page.                                                                                                                       |
 | `page[size]`   | **Optional.** If omitted, the endpoint will return 20 agent pools per page.                                                                                                              |
+| `filter[allowed_workspaces][name]` | **Optional.** Allows filtering of agent pools based on allowed workspace name. Available in Terraform Enterprise v202207-1 and later.                                |                                                                             |
```

Comment:
What do you think of this? Is this right?

```suggestion
| `filter[allowed_workspaces][name]` | **Optional.** Filters agent pools to those associated with the given workspace. The workspace must have permission to use the agent pool. Refer to [Scoping Agent Pools to Specific Workspaces](/docs/cloud-docs/agents#scoping-agent-pools-to-specific-workspaces). Available in Terraform Enterprise v202207-1 and later.                                |                                                                             |
```
```suggestion
| `filter[allowed_workspaces][name]` | **Optional.** Allows filtering of agent pools based on allowed workspace name. Available in Terraform Enterprise v202207-1 and later.                                |                                                                             |
```

Reply:
I'll update this, thanks. Should we also explicitly tell that agent pool with org scoped will also be filtered here as they are technically also accessible to the given workspace?